src/backend/agent.js

```diff
@@ -454,7 +505,7 @@ export default class Agent extends EventEmitter {
     }
   };
 
-  onHookOperations = (operations: Uint32Array) => {
+  onHookOperations = ({ operations, rendererID }: OperationsParams) => {
```

Comment:
The new `rendererID` param is redundant. The first item in the `operations` array is _always_ the renderer ID.